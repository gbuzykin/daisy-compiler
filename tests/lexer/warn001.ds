const a = 1; /* unterminated multiline comment