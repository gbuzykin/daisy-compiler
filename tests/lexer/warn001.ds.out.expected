./lexer/warn001.ds:1:1: debug: token
 1 | const a = 1; /* unterminated multiline comment
   | ^~~~~
./lexer/warn001.ds:1:7: debug: id: a
 1 | const a = 1; /* unterminated multiline comment
   |       ^
./lexer/warn001.ds:1:9: debug: token
 1 | const a = 1; /* unterminated multiline comment
   |         ^
./lexer/warn001.ds:1:11: debug: integer number: 1
 1 | const a = 1; /* unterminated multiline comment
   |           ^
./lexer/warn001.ds:1:7: debug: defining constant `a`
 1 | const a = 1; /* unterminated multiline comment
   |       ^
./lexer/warn001.ds:1:12: debug: token
 1 | const a = 1; /* unterminated multiline comment
   |            ^
./lexer/warn001.ds:1:14: warning: unterminated comment block
 1 | const a = 1; /* unterminated multiline comment
   |              ^~
./lexer/warn001.ds: info: warnings 1, errors 0
